### PR TITLE
feat(bug-1918744): Add firefox_crashreporter app to repositories.yaml

### PR DESCRIPTION
**Pull request by kik-kik**
# feat(bug-1918744): Add firefox_crashreporter app to repositories.yaml

This is in order to resolve: https://bugzilla.mozilla.org/show_bug.cgi?id=1915732

### CODE REVIEW

File: repositories.yaml [hunk 1]
```diff
@@ -403,6 +403,32 @@ applications:
         expiration_policy:
           delete_after_days: 1130
 
+  - app_name: firefox_crashreporter
+    canonical_app_name: Firefox Crash Reporter
+    app_description: >-
+      The Firefox desktop crash reporter client.
+    url: https://github.com/mozilla/gecko-dev
+    notification_emails:
+      - chutten@mozilla.com
```
Comment (badboy) [inline repositories.yaml:412]: This should be Alex' email, he's the one responsible in the end.

File: repositories.yaml [hunk 2]
```diff
@@ -403,6 +403,32 @@ applications:
...
+    channels:
+      - v1_name: firefox-crashreporter
+        app_id: firefox.crashreporter
+    moz_pipeline_metadata_defaults:
+      expiration_policy:
+        delete_after_days: 400
+    moz_pipeline_metadata:
+      crash:
+        expiration_policy:
+          delete_after_days: 1130
```
Comment (badboy) [inline repositories.yaml:430]: Why the difference here?
(Impl-wise: this crashreporter won't actually send internal pings, so the only existing ping _is_ the crash ping)

**Review by badboy — CHANGES_REQUESTED** [review]

File: repositories.yaml (hunk 2 above)
Comment (afranchuk) [reply repositories.yaml:430]: The crashreporter may send some other pings in the future (though perhaps unlikely on a long-term basis; probably only for experiments).

Comment (badboy) [reply repositories.yaml:430]: fine with keeping this then.

**Review by badboy — APPROVED** [review]
```
python3 -m probe_scraper.runner --dry-run --cache-dir tmp/cache --out-dir tmp/out --glean --output-bucket=gs://probe-scraper-prod-artifacts/ --glean-repo glean-core --glean-repo firefox-crashreporter --glean-limit-date 2024-09-17
```

runs clean.

**Pull request merged**